### PR TITLE
minor fix to ftl sdk including folder with wrong case

### PR DESCRIPTION
**Pull request by sslivins**
minor issue i came across in the cmake file, since im new to this i wanted to make sure I didnt miss something

### CODE REVIEW

**Comment by sslivins** [comment]
fixed crash, VS also wouldnt compile file that used volatile keyword...i'm going to dig into that more

**Comment by MattJustMatt** [comment]
CC @NCommander

**Comment by Doridian** [comment]
As per discussion in DM: Needs the browser plugin back in.
LGTM otherwise.

File: plugins/obs-ffmpeg/obs-ffmpeg-output.c [hunk 1]
```diff
@@ -1117,6 +1133,7 @@ static int try_connect(struct ffmpeg_output *output)
 	const char *full_streamkey;
 
 	settings = obs_output_get_settings(output->output);
+	memset(&config, 0, sizeof(config));
```
Comment (sslivins) [inline plugins/obs-ffmpeg/obs-ffmpeg-output.c:1136]: this actually fixed the crash i was experiencing as the video settings wasnt null and would later cause the application to crash when it tried to parse them

**Pull request merged**